script/release/DSL-scripts/KIE_Release.sh

```diff
@@ -511,10 +513,11 @@ job("KIE-updateToNextDevelopmentVersion-7.0.x") {
   description("This job: <br> updates the KIE repositories to a new developmenmt version <br> for 6.4.x, 6.5.x or 7.0.x branches <br> IMPORTANT: Created automatically by Jenkins job DSL plugin. Do not edit manually! The changes will get lost next time the job is generated.")
  
   parameters {
-    stringParam("newVersion", "new KIE version", "Edit the next KIE development version")
-    stringParam("UBERFIRE_VERSION", "uberfire version", "Edit the next uberfire development version")
-    stringParam("DASHBUILDER_VERSION", "dashbuilder version", "Edit the next dashbuilder development version")
-    stringParam("ERRAI_VERSION", "errai version", "Edit the next errai development version")
+    stringParam("BASE_BRANCH","master","Branch you want to upgrade")
+    stringParam("newVersion", "new KIE version", "Edit the KIE development version")
```

Comment:
I guess it would be nice to stick to a single naming scheme. Either NEW_VERSION or uberfireVersion.